app.js

```diff
@@ -0,0 +1,44 @@
+var createError = require('http-errors');
```

Comment:
TODO: next story: convert to typescript, no JS

Reply:
Created Issue for converting to typescript.